From 1512a197ad099a73787624d35f5e882f3b0ee4cf Mon Sep 17 00:00:00 2001
From: chewitt <github@chrishewitt.net>
Date: Fri, 26 Apr 2019 08:47:22 +0000
Subject: [PATCH 178/186] WIP: arm64: dts: meson-gxm-vega-s96: set rc-vega-s95
 ir keymap

S96 uses the same remote as the S95 series devices.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 arch/arm64/boot/dts/amlogic/meson-gxm-vega-s96.dts | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/amlogic/meson-gxm-vega-s96.dts b/arch/arm64/boot/dts/amlogic/meson-gxm-vega-s96.dts
index e2ea6753263b..c25fd80b14bf 100644
--- a/arch/arm64/boot/dts/amlogic/meson-gxm-vega-s96.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-gxm-vega-s96.dts
@@ -35,3 +35,10 @@
 		reg = <0>;
 	};
 };
+
+&ir {
+	status = "okay";
+	pinctrl-0 = <&remote_input_ao_pins>;
+	pinctrl-names = "default";
+	linux,rc-map-name = "rc-vega-s95";
+};
-- 
2.17.1

